flink-contrib/flink-storm/src/test/java/org/apache/flink/storm/wrappers/WrapperSetupHelperTest.java

```diff
@@ -186,15 +186,15 @@ public void testCreateTopologyContext() {
 				.shuffleGrouping("bolt2", TestDummyBolt.groupingStreamId)
 				.shuffleGrouping("bolt2", TestDummyBolt.shuffleStreamId);
 
-		LocalCluster cluster = new LocalCluster();
-		Config c = new Config();
-		c.setNumAckers(0);
-		cluster.submitTopology("test", c, builder.createTopology());
-
-		while (TestSink.result.size() != 8) {
-			Utils.sleep(100);
-		}
-		cluster.shutdown();
+//		LocalCluster cluster = new LocalCluster();
```

Comment:
OK, I open new issue in [FLINK-5522](https://issues.apache.org/jira/browse/FLINK-5522)

The reason is: Local Cluster can't run with powermock.